www/js/lib/uiUtil.js

```diff
@@ -406,9 +406,14 @@ define(rqDef, function(settingsStore) {
      * Removes the animation effect between various sections
      */
     function removeAnimationClasses() {
-        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
-        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
-        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+        document.getElementById('about').classList.remove('slideIn_L');
+        document.getElementById('about').classList.remove('slideOut_R');
+        document.getElementById('configuration').classList.remove('slideIn_L');
+        document.getElementById('configuration').classList.remove('slideIn_R');
+        document.getElementById('configuration').classList.remove('slideOut_L');
+        document.getElementById('configuration').classList.remove('slideOut_R');
+        document.getElementById('articleContent').classList.remove('slideIn_R');
+        document.getElementById('articleContent').classList.remove('slideOut_L');
```

Comment:
I think in this case it's OK to leave this as you've written it, because defining a variable would not make the code more concise.